File: backend/src/main/java/com/g4ts/agendaapp/model/Recordatorio.java
package com.g4ts.agendaapp.model;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@Table(name = "Recordatorio")
@Getter
@Setter
@ToString
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Recordatorio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private Integer idUsuario;
    private String titulo;
    private String descripcion;
    private LocalDate horaFecha;
    private Integer estado;

}

File: frontend/src/app/model/recordatorio.ts
import { Usuario } from "./usuario";

export class Recordatorio {

    id:number;
    titulo:string;
    descripcion:string = "";
    fecha:string = ""; 
    estado:number = 0;
    usuario:Usuario;

    constructor(){
        this.usuario = new Usuario();
    }
    
}